bucket/danser-go.json

```
@@ -0,0 +1,22 @@
+{
+  "version": "0.6.7",
+  "description": "Dancing visualizer of osu! standard maps and custom osu! client written in Go. Also a generator for osu! videos.",
+  "homepage": "https://github.com/Wieku/danser-go",
+  "license": "MIT",
+  "architecture": {
+    "64bit": {
+      "url": "https://github.com/Wieku/danser-go/releases/download/0.6.8/danser-0.6.8-win.zip#dl.7z",
```

```suggestion
      "url": "https://github.com/Wieku/danser-go/releases/download/0.6.8/danser-0.6.8-win.zip",
```

Already an archive, so not required